swap_meet/vendor.py

```diff
@@ -0,0 +1,58 @@
+class Vendor:
+    def __init__(self, inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory 
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else: 
+            self.inventory.remove(item)
+            return item
```

Comment:
The else isn't needed here

```suggestion
        self.inventory.remove(item)
        return item
```